go-backend/db/user.go

```diff
@@ -6,17 +6,123 @@ import (
 	logger "github.com/sirupsen/logrus"
 )
 
+const (
+	getUserQuery = `SELECT id,
```

Comment:
you should only allow to fetch user profile where soft_deleted is null. Means the user is not deleted. This goes for all queries which are made on user model